src/alloc/mod.rs

```diff
@@ -15,15 +15,15 @@ use getrandom::getrandom;
 
 const GARBAGE_VALUE: u8 = 0xd0;
 const CANARY_SIZE: usize = 16;
-static ALLOC_INIT: Once = Once::new();
-static mut PAGE_SIZE: usize = 0;
+pub(crate) static ALLOC_INIT: Once = Once::new();
+pub(crate) static mut PAGE_SIZE: usize = 0;
```

Comment:
ALLOC_INIT should probably be called PAGE_SIZE_INIT and put in a separate module

Reply:
Sure, I can move it to an `init` module or something like that. Are you thinking of adding anything else?